Command/DeclareCommand.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace OldSound\RabbitMqBundle\Command;
+
+use OldSound\RabbitMqBundle\Declarations\DeclarationsRegistry;
+use OldSound\RabbitMqBundle\Declarations\Declarator;
+use OldSound\RabbitMqBundle\RabbitMq\DynamicConsumer;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidOptionException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+class DeclareCommand extends Command
+{
+    use ContainerAwareTrait;
+
+    public function __construct(
+        DeclarationsRegistry $declarationsRegistry
+    ) {
+        parent::__construct();
+        $this->declarationsRegistry = $declarationsRegistry;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('rabbitmq:declare')
```

Comment:
please keep when possible the existing commands for backwards compatibility

Reply:
it not make sense. I am completely rework design and configurations. 
We have already a lot of break changes.
1. redesign yaml config
2. simplify design and removed `rabbitmq:rpc-server` `rabbitmq:dynamic-consumer` `rabbitmq:batch:consumer` `rabbitmq:anon:consumer` commands in favor to common one `rabbitmq:consumer`
3. remove `AmqpBase` `BatchConsumer` (`BatchExecuteReceiverStrategy` instead) `DynamicConsumer` `AnonConsumer` in favor one `Consumer`. `RcpClient` would be remove in favor to `ReplyReceiverExecutor`
4. Introduce `ExecuteReceiverStrategyInterface`
5. `BatchConsumerInteface` was removed. `ConsumerInterface` as well to favor non-interface callback `callback(AmqpMessage $message): FlagEnum` (consider consumer callback as symfony http controller which accept Request as first argument only by default)
6. multi ack for batch consumer
7. TODO annotation for declare consumer callback like 
```php
class OrderReceiver
{
  /**
   * @Consume(queue="finished_payment", connection="default", "qos": ["prefetch_count": 10], "ack": true)
   */
   public function finish(AmqpMessage $message) {
       return Consumer::FLAG_ACK;
   }

  /**
   * @Consume(queue="new_order", connection="default", "qos": ["prefetch_count": 10], "ack": true)
   * @MessageSerializer(type=OrderDTO::class)
   */
   public function sendNotification(OrderDTO $order) {
       return Consumer::FLAG_ACK;
   }
}
``` 
8. TODO $message interceptor for ability transform message and receive DTO object argument in callback